src/order/boolean_algebra.lean

```diff
@@ -706,11 +706,11 @@ theorem le_compl_iff_le_compl : y ≤ xᶜ ↔ x ≤ yᶜ :=
 theorem compl_le_iff_compl_le : xᶜ ≤ y ↔ yᶜ ≤ x :=
 ⟨compl_le_of_compl_le, compl_le_of_compl_le⟩
 
-theorem disjoint_iff_le_compl_right : disjoint x y ↔ x ≤ yᶜ :=
-by {rw is_compl.disjoint_left_iff is_compl_compl}
+lemma le_compl_iff_disjoint : x ≤ yᶜ ↔ disjoint x y := is_compl_compl.le_right_iff
```

Comment:
What is the reason for switching the order of these? 
Also, I think the names `le_compl_right` and `le_compl_left` are more descriptive than just adding a `'`

Reply:
They were introduced simultaneously in #14508, which I didn't have time to comment on. I'm keeping my version because this direction is more normalizing.

Reply:
Can you clarify what you mean by "more normalizing"? And what is the reason for removing semantic meaning in the names?

Reply:
What semantic meaning? If you mean `_left`/`_right`, I'm adding them back as soon as I have time. It's more normalizing because we would want to make `disjoint x y` the simp normal form, as opposed to `x ≤ yᶜ` which is not even unique because it could very well be `y ≤ xᶜ` as well.

Reply:
`disjoint x y` is also not unique, since it could be `disjoint y x`, in precisely the same way as `x ≤ yᶜ` could be `y ≤ xᶜ`.

Reply:
Okay, but do you prefer things to be normalised towards `x ≤ yᶜ` or `disjoint x y`. Actually, can we not make `disjoint x y ↔ disjoint y x` a simp lemma because simp has special support for commutativity lemmas?

Reply:
Moved to #14925. Note that the already existing corresponding `set` lemma agrees with my direction.